src/svelte/src/components/dataEditor.svelte

```diff
@@ -319,6 +320,7 @@ limitations under the License.
     on:seek={seekEventHandler}
   />
 
+  <Help />
```

Comment:
Would be cool if the desired help text could be bound into a generic Help component.

Reply:
The `Help` component is "generic" but only displays content from available `HelpInformation` types that are defined in the `AvailableHelpSections`. 

This component uses the contents of the `currentHelpSectionDisplayed` store to determine what is displayed.